style.css

```diff
@@ -145,9 +149,71 @@ a:hover {
     color: var(--white);
 }
 
+/* card images */
+
+.image-card__profile {
+    position: absolute;
+    left: 1em;
+    bottom: 1.5em;
+    color: var(--gray-1);
+    text-decoration: none;
+}
+
+.gallery-column_image svg {
+    width: 1.2em;
+    fill: var(--primary-gray);
+}
+
+.gallery-column button {
```

Comment:
This button is related to the image, not the column. You should style it basde on the image class, like so:

`gallery-column_image button`